github-runners/k8s/dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:buster-slim
+
+ENV GITHUB_PAT ""
+ENV GITHUB_TOKEN ""
+ENV GITHUB_OWNER ""
+ENV GITHUB_REPOSITORY ""
+ENV RUNNER_WORKDIR "_work"
+ENV RUNNER_LABELS ""
+ENV ADDITIONAL_PACKAGES ""
+
+RUN apt-get update \
+    && apt-get install -y \
+        curl \
+        sudo \
+        git \
+        jq \
+        iputils-ping \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && useradd -m github \
+    && usermod -aG sudo github \
+    && echo "%sudo ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+
+USER github
+WORKDIR /home/github
+
+RUN GITHUB_RUNNER_VERSION=$(curl --silent "https://api.github.com/repos/actions/runner/releases/latest" | jq -r '.tag_name[1:]') \
+    && curl -Ls https://github.com/actions/runner/releases/download/v${GITHUB_RUNNER_VERSION}/actions-runner-linux-x64-${GITHUB_RUNNER_VERSION}.tar.gz | tar xz \
+    && sudo ./bin/installdependencies.sh
+
+COPY --chown=github:github entrypoint.sh runsvc.sh ./
+RUN sudo chmod u+x ./entrypoint.sh ./runsvc.sh
```

Comment:
Is this the only reason we add sudo? If so this step should happen before we change user. Also I'm not quite sure while file needs to be owned by `github` user, is that user really supposed to modify the file it is running with? Otherwise default permissions should still allow to run it (if you commit `entrypoint.sh` into the repo with `+x` permissions). This looks really unnecessary to me.

Reply:
Thats the official github implementation, which requires running as sudo user. But I have removed the k8s support so don't need this anymore.